app/editor/src/features/content/form/PropertiesContentForm.tsx

```diff
@@ -1,42 +1,36 @@
-import {
-  Button,
-  ButtonVariant,
-  Col,
-  FormikCheckbox,
-  FormikDropdown,
-  FormikText,
-  FormikTextArea,
-  OptionItem,
-  RadioGroup,
-  Row,
-  SelectDate,
-} from 'components';
+import { Button, ButtonVariant } from 'components/button';
+import { Col } from 'components/col';
+import { IOptionItem, OptionItem, RadioGroup, SelectDate } from 'components/form';
+import { FormikCheckbox, FormikDropdown, FormikText, FormikTextArea } from 'components/formik';
+import { Row } from 'components/row';
 import { useFormikContext } from 'formik';
-import { IContentApi, useApiEditor } from 'hooks';
-import { useEffect, useState } from 'react';
+import React from 'react';
+import { useLookup } from 'store/hooks';
 
 import { expireOptions, summaryOptions, toningOptions } from './constants';
+import { IContentForm } from './interfaces';
 
 export interface IContentSubForms {
-  setContent: (content: IContentApi) => void;
-  content: IContentApi;
+  setContent: (content: IContentForm) => void;
```

Comment:
We will always have both `I*Form` and `I*Model` interfaces when interacting with a form and the api.  Never mix them.  All the api models will reside near their hooks.